src/renderer/components/Dex.vue

```diff
@@ -112,6 +112,14 @@ export default {
       /* eslint-disable max-len */
       this.$services.alerts.success(`${(message.side === 'bid' ? 'Buy' : 'Sell')} Order Filled. x${message.data.quantity} @${message.data.price}`);
     };
+
+    this.$store.state.socket.orderCreationFailed = (message) => {
+      this.$services.alerts.error(`Failed to Create Order ${(message.side === 'bid' ? 'Buy' : 'Sell')}. {message.errorMessage}`);
+    };
+
+    this.$store.state.socket.orderMatchFailed = (message) => {
```

Comment:
These need to be properly set using mutations. We cannot set store state directly.